### DISCUSSION TITLE
New Project

### DISCUSSION THREAD

**Discussion by VishnuChebolu** [Robotics-Club-BMU/Ardutronics#19] [General]
If you want to do a new project our your desired topic in Arduino+Electronics, Create a new issue for that or you can comment in this discussion by tagging me.

Go ahead with your PR and contributions :)

**Comment by ShubhamJagtap2000** [comment]
Hello @VishnuChebolu, I wanted to add some projects that I worked on during my college period, should I do so??

**Comment by VishnuChebolu** [comment]
Yes , of course.

Create a PR. I'll review it and merge.

**Reply by ShubhamJagtap2000** [reply]
Okay. Will create PRs in next week.

**Reply by ShubhamJagtap2000** [reply]
Also, I have projects on Computer Vision also. Shall I add them in your OpenCV repo??

**Comment by VishnuChebolu** [comment]
Yes, go ahead.
